src/main/java/mate/controller/GetCarByIdController.java

```diff
@@ -0,0 +1,33 @@
+package mate.controller;
+
+import java.io.IOException;
+import java.util.List;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class GetCarByIdController extends HttpServlet {
```

Comment:
you didn't need to implement a such controller, remove it

Reply:
You have already a such controller AddDriverToCarController. You don't need this one

Reply:
Got it already.